docs/source/guides/libraries.rst

```diff
@@ -301,12 +272,313 @@ Python Library
 Where to Get
 ^^^^^^^^^^^^
 
+Prerequisites
+"""""""""""""
+
+- CMake(3.11 or higher)
```

Comment:
Please add missing guides to dependencies.rst and place corresponding references here. Use Android Library documentation as an example (this pull request contains the updated version of Android documentation).